Assets/Scripts/Enemy & NPC/Goomba_Enemy.cs

```diff
@@ -0,0 +1,59 @@
+﻿using GGJ2021;
+using GGJ2021.NPC;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Goomba_Enemy : NPC_Base, IContactDamager ,IMobileActor<Vector3> , IHittable<int>
+{
+    /*
+     Interfaces
+     */
+    public override void hit()
```

Comment:
Why are all the function names lowercase? .NET conventions capitalizes the first letter. Other than that, looks good to me.